lib/jekyll/assets/proxies/magick.rb

```diff
@@ -171,5 +171,32 @@ def magick_preset_resize(img, cmd)
     # rubocop:enable Style/ParallelAssignment
     # rubocop:enable Metrics/AbcSize
     # ------------------------------------------------------------------------
+
+  end
+
+  Jekyll::Assets::Env.liquid_proxies.add :magick, :img, *(args + presets) do
```

Comment:
Thanks for making those changes! Updated the PR + fix in that new support changes.